### ISSUE TITLE
Protocol versioning

### ISSUE THREAD

**Issue by parkan** [mediachain/mediachain#156]
TODO: versioning at libp2p level

**Comment by vyzo** [comment]
I don't think that meta-schema versioning can be strict; we can't retroactively change extant records.

**Comment by parkan** [comment]
edited to remove no longer relevant layers
